raiden/raiden_service.py

```diff
@@ -539,6 +530,35 @@ def _start_transport(self, chain_state: ChainState) -> None:
             if neighbour != ConnectionManager.BOOTSTRAP_ADDR:
                 self.start_health_check_for(neighbour)
 
+    def _prepare_and_execute_alarm_first_run(self, from_block) -> None:
+        """Prepares the alarm task callback and executes its first run
+
+        Complete the first_run of the alarm task and synchronize with the
+        blockchain since the last run.
+
+         Notes about setup order:
+         - The filters must be polled after the node state has been primed,
+           otherwise the state changes won't have effect.
+         - The alarm must complete its first run before the transport is started,
+           to reject messages for closed/settled channels.
+        """
+        assert not self.transport, f"Transport is running. node:{self!r}"
```

Comment:
So, the asserts I had in mind were the two from the docstring:

- the node state has been primed
- before the transport is started

Reply:
Hmm I put the asserts you asked in your last [comment](https://github.com/raiden-network/raiden/pull/4550#discussion_r311620861).

Apart from the second one since the alarm task needs to be primed after the first run.

> the node state has been primed

What do you mean by that? That the DB/wal is initialized?